docs/_articles/en/v3.x/faq.md

```diff
@@ -2,106 +2,114 @@
-A: When user runs the code analyzer with the `-s` or `--severity-threshold` flag and a threshold value, the code analyzer throws an error if violations are found with equal or greater severity than provided value. Values are 1 (high), 2 (moderate), and 3 (low). Exit code is the severity of the most severe violation(s). Using this flag also implicitly invokes the --normalize-severity flag.
+#### Q: How do I add new rules for language X?
+A: Currently, you can only add custom rules for PMD. 
 
-#### Q: How do I get normalized severity?
-A: PMD, ESLint & RetireJS all have different scales for reporting the Severity of the violations. When the user runs the code analyzer with the `--normalize-severity` flag, the `Salesforce Code Analyzer` will normalize the severity of violations across all invoked engines.  A normalized severity 1 (high), 2 (moderate), and 3 (low) is returned in addition to the engine specific severity. For the html output format, the normalized severity is displayed instead of the engine severity	
+* Ruleset files, and category files defining only XPath-based rules, are referenced as standalone XML files. 
+* Java-based rules must be bundled into a JAR. The JAR/XML can then be added to the rule registry with the `scanner:rule:add` command.
 
-#### Q: How is the Severity normalized across all the engines?
-A: Following table shows how the severity across all engines are normalized. 
+If the language isn’t already supported, create an issue on our [Github repo](https://github.com/forcedotcom/sfdx-scanner).
+
+### Questions about Severity Thresholds and Normalization
+
+#### Q: How do I set a severity threshold for a Code Analyzer run?
+A: When you run Code Analyzer with the `-s | --severity-threshold` flag and a threshold value, Code Analyzer throws an error if violations are found with equal or greater severity than the provided value. Values are 1 (high), 2 (moderate), and 3 (low). The exit code equals the severity of the most severe violation detected. For example, if a violation of severity 2 is found and the threshold is 2 or 3, then the exit code is 2. Using this flag also implicitly invokes the `--normalize-severity` flag.
+
+#### Q: How do I normalize severity?
+A: PMD, ESLint, and RetireJS all have different scales for reporting the severity of violations. When you run Code Analyzer with the `--normalize-severity` flag, Code Analyzer normalizes the severity of violations across all invoked engines. 
+* A normalized severity of 1 (high), 2 (moderate), and 3 (low) is returned in addition to the engine specific severity. Applicable engines include: 
+	- CSV
+	- HTML
+	- JSON
+	- JUnit-formatted XML
+	- SARIF-formatted JSON
+	- Table (default output)
+	- XML
+* For HTML output format, the normalized severity is displayed instead of the engine severity.
+
+#### Q: How is the severity normalized across all the engines?
+A: Severity is normalized across all engines using the values in this table.
```

Comment:
Does SFGE need an entry in this table? On the one hand, its normalized severities are identical to its non-normalized severities so the data is redundant. On the other hand, its absence is conspicuous.